atd-vze/src/routes.js

```diff
@@ -161,6 +162,12 @@ const routes = roles => [
     name: "Crash Details",
     component: Crash,
   },
+  {
```

Comment:
This path needs one more protection to keep read-only and editors from visiting it. Line 189 in here has some role checks as an example. 

The good thing is that the query fails if you visit this page as a read-only or editor thanks to the Hasura permissions 🙌 , but adding the role checks will redirect to the dashboard view if anyone types the path in the address bar.

Also, I had forgotten they existed, but there are some test accounts in 1Password for each role that are called **VZE Test User: Editor**, etc if you ever need them. They came in handy for me to test this. 🧰

Reply:
oo yeah i need to use the test user more option. thanks for reminding me about it!